README_extending_docker.md

```diff
@@ -82,15 +59,83 @@ You can use our base docker to extend the functionality to fit your need. The st
        $CUSTOM_DOCKER_NAME:$CUSTOM_DOCKER_VERSION 
    ```
 
-   This would run your `custom docker` and you can point your browser to `http://127.0.0.1:8888/lab/tree/Welcome.ipynb`! 
-   Audit logs will not be persisted across docker deletion. In order to persist them, you can use the following command (assuming the data needs to be stored in $HOME/data)
+   This would run your `custom docker` and you can point your browser to `http://127.0.0.1:8888/awesome`! 
 
-   ```
-   docker run -it -p 8888:8888 \
-       -v $HOME/data:/data  \
-       $CUSTOM_DOCKER_NAME:$CUSTOM_DOCKER_VERSION 
-   ```
+2. Push your `custom docker` to any docker registry for redistribution.
+<br/>
+
+
+
+## How to Copy Custom Actions and Runbook 
+
+If you have deployed our Awesome runbook as a Kubernetes POD then follow the step below
+1. Copy the custom actions from the POD to your local machine so you can bundle into your custom Docker for re-distribution
+```
+kubectl cp <AWESOME_POD_NAME>:/unskript/data/runbooks -n <NAMESPACE> $YOUR_REPO_DIRECTORY/runbooks
+kubectl cp <AWESOME_POD_NAME>:/unskript/data/actions -n <NAMESPACE> $YOUR_REPO_DIRECTORY/actions
+cd $YOUR_REPO_DIRECTORY
+git clone https://github.com/unskript/Awesome-CloudOps-Automation.git
+
+Example:
+
+kubectl cp awesome-runbooks-0:/unskript/data/actions -n awesome-ops $YOUR_REPO_DIRECTORY/actions 
+kubectl cp awesome-runbooks-0:/unskript/data/runbooks -n awesome-ops $YOUR_REPO_DIRECTORY/runbooks
+```
+
+If you have deployed our Awesome runbook as a Docker instance, then you can use
+the following step.
+```
+export CONTAINER_ID=`docker ps | grep awesome-runbooks | awk '{print $1}'`
+docker cp $CONTAINER_ID:/unskript/data/actions $HOME/Workspace/acme/actions
+docker cp $CONTAINER_ID:/unskript/data/runbooks $HOME/Workspace/acme/runbooks
+```
+
+Lets consider two scenarios as starting point for extending Awesome docker
+
+### How to create Custom Actions
+
+#### Steps for awesome-runbooks deployed as K8S POD
```

Comment:
Create custom actions should have been covered somewhere already, right? if not, just mention the export functionality, doesnt matter how it was launched

Reply:
Done